src/main/java/ludogorie_soft/reservations_platform_api/entity/Property.java

```diff
@@ -0,0 +1,32 @@
+package ludogorie_soft.reservations_platform_api.entity;
+
+import jakarta.persistence.*;
+import lombok.Data;
+
+import java.util.List;
+
+@Entity
+@Data
+public class Property {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne
+    @JoinColumn(name = "user_id")
+    private User owner;
+
+    private String name;
+    private String type;
+
+    private String airBnbUrl;
+    private String bookingUrl;
+
+    private String syncUrl;
```

Comment:
Can we think of even more descriptive name - for example `iCalSyncUrl` or smth else

Reply:
Done!